# HG changeset patch
# User Rik <rik@octave.org>
# Date 1521662681 25200
#      Wed Mar 21 13:04:41 2018 -0700
# Branch stable
# Node ID fb8d10420a75ff94172d34469572a216106c42f4
# Parent  bff4a7c7bc393fcc4250d83730af662b39f4e199
mk_qthelp.pl: Remove unused module autodie.

* mk_qthelp.pl: Remove unused module autodie.

diff --git a/doc/interpreter/mk_qthelp.pl b/doc/interpreter/mk_qthelp.pl
--- a/doc/interpreter/mk_qthelp.pl
+++ b/doc/interpreter/mk_qthelp.pl
@@ -3,17 +3,16 @@
 ################################################################################
 # File    : mk_qthelp.pl
 # Purpose : Transform Octave HTML documentation into intermediate formats
 #           for Qt Help Project (.qhp) and Qt Help Collection Project (.qhcp).
 # Usage   : mk_qthelp.pl input_htmldir output_filename 
 ################################################################################
 use warnings;              # report warnings for questionable run-time code
 use strict qw(refs subs);  # check at compile-time for bad programming style
-use autodie;               # issue an error and exit if any system call fails
 use File::Basename;        # For splitting paths between dir and file
 use File::Spec;            # For combining dirs and filenames into paths
 
 ################################################################################
 # Extract command line arguments
 if ($#ARGV != 1)
 { die "USAGE: %0 input_htmldir output_filename"; } 
 
